lib/internal/bootstrap/loaders.js

```diff
@@ -270,29 +259,29 @@
       const cache = codeCacheHash[id] &&
         (codeCacheHash[id] === sourceHash[id]) ? codeCache[id] : undefined;
 
-      // (code, filename, lineOffset, columnOffset
-      // cachedData, produceCachedData, parsingContext)
-      const script = new ContextifyScript(
-        source, this.filename, 0, 0,
-        cache, false, undefined
+      const fn = compileFunction(
+        source,
+        this.filename,
+        0,
+        0,
+        cache,
+        false,
+        undefined,
+        [],
+        ['exports', 'require', 'module', 'process', 'internalBinding']
```

Comment:
I'd prefer we use a variable to refer to the function arguments somehow before passing them into the `compileFunction`, for better readability (I like that the the previous `NativeModule.wrapper` is self-explanatory)...or put a comment for each one for these arguments?

Reply:
:+1: Yep, I had to lookup the args again when checking it 🙃

Reply:
Missing an comment about the last argument here?